docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,222 @@
+.. currentmodule:: discord
+
+.. _guide_topic_permissions:
+
+
+Permissions
+============
+
+Permissions are the bread and butter of how control over a Discord server (which will from now on be referenced as a "guild" to align with Discord's official API terminology).
+
+Controlling permissions is necessary for securing and controlling how your guild functions and operates.
+
+
+Permission hierarchy explanation
+---------------------------------
+
+Discord permissions work on a hierarchy system. This means that roles placed logically above each other (in the Roles UI) have descending order of precedence.
+
+If you have a role list, like so:
+
+- ``Administrator (#1)``
+- ``Manager (#2)``
+- ``Moderator (#3)``
+- ``@everyone (#4)``
+
+This means that Administrator role (someone who has it) can kick, ban and timeout Manager.
+Manager **cannot** kick, ban or timeout Administrator.
```

Comment:
Minor nit-pick here but L19 and L26 don't seem to flow very well. Also unsure about the wording on L26. 

```suggestion
Say you have a role list, like so:

- ``Administrator (#1)``
- ``Manager (#2)``
- ``Moderator (#3)``
- ``@everyone (#4)``

Here, someone with the Administrator role can kick, ban and timeout Manager.
Manager **cannot** kick, ban or timeout Administrator.
```